test/integration/controller/jobs/jobset/jobset_controller_test.go

```diff
@@ -1051,3 +1051,132 @@ var _ = ginkgo.Describe("JobSet controller interacting with scheduler", ginkgo.O
 		})
 	})
 })
+
+var _ = ginkgo.Describe("JobSet controller preemption tests for workloads", ginkgo.Ordered, ginkgo.ContinueOnFailure, func() {
```

Comment:
What is the motivation that we need to prepare another `Describe` table?
This another `Describe` table bring us longer testing duration.

In general, we have a separate `Describe` table when we need to prepare another KueueConfig setup there `fwk.RunManager(cfg, managerSetup(jobframework.WithManageJobsWithoutQueueName(true)))`.

Through the test cases, I don't think this separate `Describe` table. Could you include new test cases in the existing table?

https://github.com/kubernetes-sigs/kueue/blob/e98d0abbcc41646adc906001a70f7643531e0969/test/integration/controller/jobs/jobset/jobset_controller_test.go#L55

Reply:
Indeed, as you say, it would be better to incorporate the process I wrote into an existing table.